.github/workflows/integration.yml

```diff
@@ -77,26 +88,15 @@ jobs:
             localhost:5000/streamshub/console-ui:${{ env.PROJECT_VERSION }}
 
       # ==================== API & Operator ====================
-      - name: Test Projects and Build API and Operator Images
-        env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-          SONAR_ORG: ${{secrets.SONAR_ORG}}
-          SONAR_PROJECT: ${{secrets.SONAR_PROJECT}}
-          SONAR_TOKEN: ${{secrets.SONAR_TOKEN}}
+      - name: Build API and Operator Images
         run: |
-          #
-          # Set write-transformed-bytecode-to-build-output for IT coverage. Do NOT use the container image
-          # created by this step.
-          #
-          # See: https://quarkus.io/guides/tests-with-coverage#coverage-for-integration-tests
-          #
           export QUARKUS_CONTAINER_IMAGE_REGISTRY="localhost:5000"
           export QUARKUS_CONTAINER_IMAGE_PUSH=true
           export QUARKUS_CONTAINER_IMAGE_TAG="${{ env.PROJECT_VERSION }}"
           export QUARKUS_KUBERNETES_VERSION="${{ env.PROJECT_VERSION }}"
-          mvn verify -P container-image -B --no-transfer-progress \
+
+          mvn verify -P container-image -B --no-transfer-progress -DskipTests \
```

Comment:
Do we want to skip tests here? I know it saves a lot of time and i see you have added parallel test-java to workflow, so it might be alright, but will that use more GH storage if the build is okay, but tests fail? I think the saving images task was skiped when the test failed so it did not produce any artifacts. If this is not a problem, then i think we can merge. Thanks!

Reply:
@jankalinic I think it's OK. The storage is temporary and Github deletes it after a while anyway. We could also add the `test-java` job to the `needs` of the Playwright step, but that would still use the storage to upload the images.